base/inference.jl

```diff
@@ -3916,16 +3908,27 @@ function inlineable(f::ANY, ft::ANY, e::Expr, atypes::Vector{Any}, sv::Inference
         rettype = linfo.rettype
     end
 
+    inferred === nothing && return invoke_NF(argexprs0, e.typ, atypes, sv,
+                                        atype_unlimited, invoke_data)
```

Comment:
Kinda weird indentation here. `atype_unlimited` should probably be aligned beneath `argexprs0` for consistency with the `invoke_NF` call a few lines down.